desafio-07/kortkamp/c/tac.c

```diff
@@ -0,0 +1,93 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdio.h>
+#define LINE_BUFFER_SIZE 100000 
+#define BUFFER_SIZE 1000000
+
+/* TODO LIST	
+ * -need to deal with lines bigger then LINE_BUFFER_SIZE(yes, im noob)
+ * -print the line using the data stored in buffer, not loading linebuffer
+ * -use malloc instead of arrays 
+ * 
+ *
+ */
+
+FILE *fptr;
```

Comment:
Essa variável deveria ser declarada dentro de main, e passada pra quem precisa ler o arquivo (se for o caso). Idem pra buffer e linebuffer (abaixo).

Reply:
feito